src/pseudopeople/noise_functions.py

```diff
@@ -416,7 +415,20 @@ def use_fake_names(
     column = data[column_name]
     fake_first = fake_first_names
     fake_last = fake_last_names
-    fake_names = {"first_name": fake_first, "last_name": fake_last}
+    fake_names = {
```

Comment:
I'd recommend removing the usage of naked strings. Could do this instead:
```
fake_names = {
    "COLUMNS.first_name.name: fake_first,
    "COLUMNS.last_name.name: fake_last,
    "COLUMNS.spouse_first_name.name: fake_first,
    ...
}
```

Reply:
I tried to use local imports to avoid the circular dependencies that prevents being able to do this but it did not work.